### ISSUE TITLE
Domain Isolated Web Parts don't appear anymore

### ISSUE THREAD

**Issue by mbratukha** [SharePoint/sp-dev-docs#4788]
#### Category
- [ ] Question
- [ ] Typo
- [x] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior

The Domain Isolated Web Parts work without errors.

#### Observed Behavior

Yesterday a new version of our several Domain Isolated Web Parts was deployed to a customer production tenant. The deployment was without any errors and the Web Parts worked as expected. After several hours they just stopped to work in all browsers without any reason. One another Domain Isolated Web Part that was deployed one week before is also started not to work. We have three independent Site Collections where all these Web Parts are used. The only one is affected with this issue, i.e. it seems like this is not a tenant issue.

Today I've deployed a fresh new Hello World Domain Isolated Web Part ([see here](https://github.com/mbratukha/Hello-World-Domain-Isolated-Web-Part)) to the same tenant. This Web Part does not work **only** on the affected site collection.

Errors in Chrome (v. 77.0.3865.120, 64-bit):
![image](https://user-images.githubusercontent.com/1413357/67123543-ebaded00-f1f0-11e9-910a-d3029dba5203.png)

Errors in Firefox (v. 71.0b2, 64-bit):
![image](https://user-images.githubusercontent.com/1413357/67123697-59f2af80-f1f1-11e9-8fa7-22f78a535422.png)

Errors in Edge (v. 18.18362):
![image](https://user-images.githubusercontent.com/1413357/67124507-ee114680-f1f2-11e9-9100-eaa21a606b4f.png)

Unfortunately, we couldn't find any solution or workaround yet.

The possibility to convert all web parts to non-isolated web parts is known, but this is not a solution in our case.

Related issues: #1750 and #3881

Any help would be very appreciated!

**Comment by patmill** [comment]
So, the CSP warnings are a red-herring, although the fact that you are seeing them in Chrome means something strange is going on (or you aren't using Chrome).  Looking at the overall metrics in the service around Isolated Parts, this seems like an isolated issue.  What is under the "Technical Details" button?  I will try creating a simple HelloWorld part on a test tenant and see if I can get this to repro as well.

**Comment by johnguy0** [comment]
@mbratukha Is the issue still occurring?

**Comment by mbratukha** [comment]
@patmill Sorry for the late answer. There is nothing special under the technical details, just a correlation ID. The issue occurred only on a single site collection. Other site collections on the same tenant didn't had this issue. We also tried to install the same WebPart on another tenants and it worked without any problems. After the long discussion with the customer, we deployed an non-isolated version of this WebPart.

@johnguy0 I've just was able to test the [hello-world WebPart](https://github.com/mbratukha/Hello-World-Domain-Isolated-Web-Part) on the affected site collection. The issue is **not** happening anymore.
